Source/Scene/PostProcessStageLibrary.js

```diff
@@ -0,0 +1,665 @@
+     * @return {PostProcessStage} A post-process stage that applies Fast Approximate Anti-aliasing to the input texture.
+     *
+     * @private
+     */
+    PostProcessStageLibrary.createFXAAStage = function() {
+        return new PostProcessStage({
+            name : 'czm_FXAA',
+            fragmentShader : fxaaFS,
+            sampleMode : PostProcessStageSampleMode.LINEAR
+        });
+    };
+
+    /**
+     * Creates a post-process stage that renders the input texture with black and white gradations.
+     * <p>
+     * This stage has one uniform value, <code>gradations</code>, which scales the luminance of each pixel.
+     * </p>
+     * @return {PostProcessStage} A post-process stage that renders the input texture with black and white gradations.
+     */
+    PostProcessStageLibrary.createBlackAndWhiteStage = function() {
+        return new PostProcessStage({
+            name : 'czm_black_and_white',
+            fragmentShader : BlackAndWhite,
+            uniforms : {
+                gradations : 5.0
+            }
+        });
+    };
+
+    /**
+     * Creates a post-process stage that saturates the input texture.
+     * <p>
+     * This stage has one uniform value, <code>brightness</code>, which scales the saturation of each pixel.
+     * </p>
+     * @return {PostProcessStage} A post-process stage that saturates the input texture.
+     */
+    PostProcessStageLibrary.createBrightnessStage = function() {
+        return new PostProcessStage({
+            name : 'czm_brightness',
+            fragmentShader : Brightness,
+            uniforms : {
+                brightness : 0.5
+            }
+        });
+    };
+
+    /**
+     * Creates a post-process stage that transforms the input texture to a pixelated eight-vit style image.
+     * @return {PostProcessStage} A post-process stage that transforms the input texture to a pixelated eight-vit style image.
+     */
+    PostProcessStageLibrary.createEightBitStage = function() {
+        return new PostProcessStage({
+            name : 'czm_eight_bit',
+            fragmentShader : EightBit
+        });
+    };
+
+    /**
+     * Creates a post-process stage that adds a night vision effect to the input texture.
+     * @return {PostProcessStage} A post-process stage that adds a night vision effect to the input texture.
+     */
+    PostProcessStageLibrary.createNightVisionStage = function() {
+        return new PostProcessStage({
+            name : 'czm_night_vision',
+            fragmentShader : NightVision
+        });
+    };
+
+    /**
+     * Creates a post-process stage that replaces the input color texture with a black and white texture representing the fragment depth at each pixel.
+     * @return {PostProcessStage} A post-process stage that replaces the input color texture with a black and white texture representing the fragment depth at each pixel.
+     *
+     * @private
+     */
+    PostProcessStageLibrary.createDepthViewStage = function() {
```

Comment:
Is this used anywhere?  I assume we want to keep it for debugging?

Reply:
Yes, I removed it from the example but it's useful for debugging.